### ISSUE TITLE
FG.DSAutoPause Overwritten on Reboot

### ISSUE THREAD

**Issue by Dorkstinn** [ghostcap-gaming/pterodactyl-eggs#2]
The FG.DSAutoPause that sets whether or not the server automatically pauses if no users are online gets overwritten upon reboot. It would be beneficial if this was an option similar to FG.NetworkQuality.

**Comment by pixelwave** [comment]
Same issue - upvote! :)

**Comment by pedrotski** [comment]
Isn't this set from the in game server manager now?

**Comment by pixelwave** [comment]
Yes and the issue is any time the server reboots (for whatever reason) the auto-pause setting is reset.

So if you want auto pause **disabled** and for example you do maintenance and reboot the game server ... you additionally have to start the game, login server manager UI and uncheck auto-pause. Every. Time.

**Comment by pedrotski** [comment]
Strange. Please make sure Validate Server Files is empty so when your server starts its not set back to default.
